File: HelloWorld/Hello.cs
//by sree - https://github.com/Phasmic
using System;

namespace HelloWorld
{
    class HelloWorld
    {
        public static void Main(string[] args)
        {
            Console.Write("Enter your name: ");

            string name = Console.ReadLine();

            Console.WriteLine($"Hello, {name}");
        }
    }
}

File: HelloWorld/Hello.cpp
//by sree - https://github.com/Phasmic
#include <iostream>

using namespace.std;

class HelloWorld
{
    int main()
    {
        char name[100];

        cout << "Enter your name: ";

        cin.getline(name, 100);

        cout << "\n hello, " << name;

        return 0;
    }
}